packages/expo-document-picker/ios/DocumentPickerModule.swift

```diff
@@ -49,7 +49,8 @@ public class DocumentPickerModule: Module, PickingResultHandler {
 
   func didPickDocumentsAt(urls: [URL]) {
     guard let options = self.pickingContext?.options,
-    let promise = self.pickingContext?.promise else {
+      let promise = self.pickingContext?.promise
+    else {
```

Comment:
```suggestion
      let promise = self.pickingContext?.promise else {
```

Reply:
How do you fix these formats? I tried 4 different formatters from the VSCode marketplace and even ran swiftlint --fix from the CLI but it's somehow not working as expected

Reply:
I just use swiftlint from the cli but it doesn't fix everything. Most of the rules you just get used to and write it that way. If you run `swiftlint --rules` it will tell you which ones it can correct

Reply:
Ok, addressed. Gotta learn those rules